assets/js/src/common/form/toggle/toggle.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { InputHTMLAttributes } from 'react';
+import classnames from 'classnames';
+
+type Props = InputHTMLAttributes<HTMLInputElement> & {
+  dimension?: 'small',
+  onCheck: (isChecked: boolean) => void,
```

Comment:
There isn't really any `off` event, you can't uncheck radio button by itself. The naming is the same as in checkboxes because `onChange` is native HTML attribute which has its limitations. But practically, single `<Radio>` component probably won't ever be used and it will be used inside a group of radio buttons. And radio group component, since it's not extending any HTML element types, has meaningful `onChange` parameter.